builtin/mainmenu/modmgr.lua

```diff
@@ -335,8 +335,8 @@ function modmgr.installmod(modfilename,basename)
 	local modpath = modmgr.extract(modfile)
 
 	if modpath == nil then
-		gamedata.errormessage = fgettext("Install Mod: file: \"$1\"", modfile.name) ..
-			fgettext("\nInstall Mod: unsupported filetype \"$1\" or broken archive", modfile.type)
+		gamedata.errormessage = fgettext("Install mod: File: \"$1\"", modfile.name) ..
```

Comment:
'Install Mod' to refer to the function 'installmod'.

Reply:
It seems that these messages are referring to the function 'modmgr.installmod()' which is possibly why it is written 'Install Mod', so how this was seems fine, it doesn't need changing.